src/decks/edit-new/EditDeck.js

```diff
@@ -1,37 +0,0 @@
-import React, { useState, useEffect } from "react";
-import DeckForm from "./DeckForm";
-import { readDeck } from "../../utils/api";
-import { useParams } from "react-router-dom";
-import BreadCrumb from "../../common/BreadCrumb";
-
-function EditDeck() {
-  const { deckId } = useParams();
-  const [editDeckData, setEditDeckData] = useState({});
-  const [deck, setDeck] = useState({});
-
-  useEffect(() => {
-    async function getDeck() {
-      try {
-        setEditDeckData(await readDeck(deckId));
-        setDeck(await readDeck(deckId));
```

Comment:
Why are we fetching the data twice here? I understand that the original deck data needs to be used for the breadcrumb, but we can store the value that we can from reading the deck in a variable in here so we don't create two network requests